smoke-test/tests/cypress/cypress/integration/search/search.js

```diff
@@ -1,65 +1,195 @@
-describe('search', () => {
-  it('can hit all entities search, see some results (testing this any more is tricky because it is cached for now)', () => {
-    cy.login();
-    cy.visit('/');
-    cy.get('input[data-testid=search-input]').type('*{enter}');
-    cy.wait(5000);
-	  cy.contains('of 0 results').should('not.exist');
-	  cy.contains(/of [0-9]+ results/);
-  });
+describe("search", () => {
+  // it("can hit all entities search, see some results (testing this any more is tricky because it is cached for now)", () => {
```

Comment:
are these meant to be commented out....?

Reply:
this has been reverted already :)